### PR TITLE
increase fuzz factor for search perf on vms

### PR DESCRIPTION
**Pull request by mjkkirschner**
### Purpose

increase timeout fuzz factor

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### CODE REVIEW

File: test/DynamoCoreTests/Search/SearchDictionaryTest.cs [hunk 1]
```diff
@@ -93,7 +93,7 @@ public void TestSearchDictionaryPerformance()
             Assert.AreEqual(results.Count(), 20);
 
             double timeLimit = 300;//ms
-            double range = 30;
+            double range = 70;
             Assert.IsTrue(Math.Abs(stopwatch.ElapsedMilliseconds - timeLimit) < range / 100 * timeLimit, $"Search time should be within a range of +/- {range}% of {timeLimit}ms but we got {stopwatch.ElapsedMilliseconds}ms");
```
Comment (aparajit-pratap) [inline test/DynamoCoreTests/Search/SearchDictionaryTest.cs:97]: So we are okay with a 70% variation from the expected time?

Comment (mjkkirschner) [reply test/DynamoCoreTests/Search/SearchDictionaryTest.cs:97]: It's hard to predict the performance of all machines this test will run on including VMs that might be on shared machines themselves.

**Comment by mjkkirschner** [comment]
passed on serial self serve - waiting for parallel tests to merge.

File: test/DynamoCoreTests/Search/SearchDictionaryTest.cs (hunk 1 above)
Comment (Dewb) [reply test/DynamoCoreTests/Search/SearchDictionaryTest.cs:97]: Perhaps we could get some kind of coarse performance metrics from the VM (like vCPU count?) and have different scaling factors for different levels of expected performance? With a wide error margin we aren't getting as much value out of running performance tests.

**Comment by mjkkirschner** [comment]
merging this to get builds running - can investigate improving this later.

**Pull request merged**